src/main/java/org/graylog2/syslog4j/server/impl/event/FortiGateSyslogEvent.java

```diff
@@ -55,7 +68,7 @@ private void parse(String event) {
             parsePriority(priority);
             setMessage(message);
             parseFields(message);
-            parseDate(fields.get("date"), fields.get("time"));
+            parseDate(fields.get("date"), fields.get("time"), fields.get("tz"));
```

Comment:
This might break some existing pipeline rules

Reply:
With https://github.com/Graylog2/graylog2-server/issues/3853 I suppose all existing workarounds for the timezones will need to be adjusted, so this needs to be clear from the changelog there.